osu.Game.Tests/Visual/TestCasePlaySongSelect.cs

```diff
@@ -62,7 +62,12 @@ private void load()
             var storage = new TestStorage(@"TestCasePlaySongSelect");
```

Comment:
This should probably be guid'd now, clicking on `TestCasePlaySongSelect` multiple times can cause crashes:

`Microsoft.Data.Sqlite.SqliteException: SQLite Error 10: 'disk I/O error'.`